library/init/data/nat/lemmas.lean

```diff
@@ -730,5 +770,148 @@ lemma pred_inj : ∀ {a b : nat}, a > 0 → b > 0 → nat.pred a = nat.pred b 
 
 /- TODO(Leo): sub + inequalities -/
 
+/- div -/
+
+theorem zero_div : ∀ m : ℕ, 0 / m = 0
+  | zero := by refl
+  | (succ n) := by refl
+
+theorem div_le_div
+: ∀ {y x}, x ≤ y → ∀ z : ℕ, x / z ≤ y / z :=
```

Comment:
I was thinking it might be useful to rewrite x /z <= y / z to x <= y.  If Lean's elaborate will already do that, then great.  I also don't think that change is critical.